docs/raw/organisation-channels.md

```diff
@@ -0,0 +1,160 @@
+---
+permalink: /spec/17
+parent: Stable specs
+title: 17/ORGANISATION-CHANNELS
+---
+
+# 17/ORGANISATION-CHANNELS
+
+> Version: 0.1
+>
+> Status: Raw
+>
+> Authors: Andrea Maria Piana <andreap@status.im>
+>
+
+- [Organisation channels](#17organisation-channels)
+  - [Reason](#reason)
+  - [Fetching information about an organisation](#fetching-information-about-an-organisation)
+  - [Joining an organisation](#joining-an-organisation)
+  - [Updating channels](#updating-channels)
+
+## Reason
+
+Organisation channels are a way to allow users to build communities and create curated content.
+
+## Fetching information about an organisation
+
+An organisation is uniquely identified by a public key.
+
+This public key can be discovered through different means, links, indexable web pages or search directories for example.
+
+Given that the public key of the organisation is known, information about the channels and members can be fetched by listening to a discovery topic derived by the public key:
+
+```
+   Does not matter which topic for now, as long as it does not clash with public chats.
+```
+
+The organisation SHOULD publish periodically an `OrganisationDescription` message,
+at least once every 24 hours, ideally more to minimise the dependency on mailservers.
+
+
+```protobuf
+message ChatMessageIdentity {
+  // stubbing this, it's already been prepared in a different PR
+  string ens_name = 1;
+  string profile_picture = 2; // not a string, more complex data structure, but you get the point
+  string display_name = 3; 
+  string description = 4; // this is actually unique to an organisation, and chatMessageIdentity might want to be updated, or similar
```

Comment:
Not sure how extensive this needs to be; I imagine it can't be updated for channels that have already been created later. Other information we might want for a sorted and categorized view of chats would be:
- date created
- category tags
- color code (for basic on brand styling)

Reply:
We can add new fields as we implement new functionalities, this is always a non-breaking change (we can't remove fields).
So for things like "Category tags"/Color code, unless we use them in the MVP, no need to add them at this stage and can always be added at a later stage (older client will ignore them, but that's expected as they would not know what to do with those fields, unless we implement the handling from the get go)

Reply:
Let me think on when we would want to add these. If the data is available, Desktop client likely starts handling them as soon as possible as they're looking for a more visual and rich way to explore community chats and other (more visual real estate, lower discovery risks wrt store policies)